third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```diff
@@ -1,7 +1,86 @@
-controllerManager:
+runtimeConfig:
+  health: {}
+    # healthProbeBindAddress: :8081
+  metrics: {}
+    # bindAddress: 127.0.0.1:8080
+  webhook: {}
+    # port: 9443
+  leaderElection: {}
+    # leaderElect: true
+    # resourceName: e29e094a.k8s.io
+  manager:
+    runtime: containerd
+    otlpEndpoint: ""
+    logLevel: info
+    scheduling: {}
+      # repeatInterval: "" # to be parsed into time.Duration
+      # beginImmediately: true
+    profile: {}
+      # enable: false
+      # port: 0
+    imageJob:
+      successRatio: 1.0 # float; ok with YAML?
```

Comment:
remove comment here too and anywhere else this applies

Reply:
done

Reply:
still seeing `# float; ok with YAML?` comment here

Reply:
done